galactic/distribution.yaml

```diff
@@ -4190,6 +4190,17 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: galactic
     status: developed
+  rospy_message_converter:
+    doc:
+      type: git
+      url: https://github.com/uos/rospy_message_converter.git
+      version: galactic
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/uos/rospy_message_converter.git
```

Comment:
Please add a LICENSE file to the source repository.

Reply:
Done!